portfolio/src/main/webapp/script.js

```diff
@@ -115,21 +115,21 @@ function viewReplies() {
 }
 
 const LOCATIONS = [
```

Comment:
nit.
IMO this would read better if this was an array of objects instead of array of arrays.
That way, when you use it below, you can do location.name instead of location[0], which isn't as obvious what's happening

Reply:
^ also if you do it with objects, you might be able to use a constructor-like function to create your objects for you.

Reply:
Sounds like a plan, changed in the 2nd most recent PR